changes/10104-policy-tab-click-bug

```diff
@@ -1 +0,0 @@
-* Fix bug to keep team when clicking on policy tab twice
```

Comment:
Should this have been deleted here?

Reply:
Yes, this is on `main`. I will include in 4.29.0 release notes.